app/src/main/java/vn/mbm/phimp/me/Upload.java

```diff
@@ -257,23 +259,24 @@ public void onItemClick(AdapterView<?> parent, View v, int position, long id) {
         });
 
 		/*
-         * bluetooth share
+         * bluetooth shareSingle
 		 */
         btnBluetoothShare = (ImageView) getView().findViewById(R.id.upload_sendDirectly);
         btnBluetoothShare.setColorFilter(color);
         btnBluetoothShare.setOnClickListener(new OnClickListener() {
             @Override
             public void onClick(View v) {
                 if (imagelist != "") {
-                    Intent intent = new Intent();
+                    ImageSharer.share(getContext(),imagelist);
+                   /* Intent intent = new Intent();
                     intent.setClass(getContext(), BluetoothShareMultipleFile.class);
                     intent.putExtra("imagelist", imagelist);
                     intent.putExtra("activityName", "Upload");
                     intent.addFlags(Intent.FLAG_ACTIVITY_CLEAR_TOP);
-                    startActivity(intent);
+                    startActivity(intent);*/
 
                 } else {
-                    Commons.AlertLog(ctx, "Do not have photo to share", getString(R.string.accept)).show();
+                    Commons.AlertLog(ctx, "Do not have photo to shareSingle", getString(R.string.accept)).show();
```

Comment:
Remove the previous code you have commented, make the logs and comments  logic. (Besure about Strings) :) @harsu-ag